packages/tools/devtools/devtools-browser-extension/package.json

```
@@ -53,8 +53,7 @@
 		],
 		"exclude-after-remap": false,
 		"include": [
-			"src/**/*.ts",
-			"dist/**/*.js"
+			"src/**/*.ts"
```

@chentong7 can you try running a pipeline with this here? (And you can revert the change to `devtools-core`. I tested locally and this accomplishes what I think we need to fix this. Basically telling c8 that the only files within `dist` that it should be checking for coverage are the ones that resulted from compiling the ones in `src`. The other folders under `dist` are for other purposes, and I'm not sure exactly why, but if we let c8 look at all of them as potential source files to be checked for coverage, it ends up generating a _ton_ of extra output under `nyc/report`, including a `node_modules` folder, which explains the huge amount of files that it's looking for and failing to find when the pipeline tries to publish the code coverage report.
```suggestion
			"src/**/*.ts"
			"dist/tsc/**/*.js"			
```

If you run `npx c8 --clean npm run test:jest` on this package locally, with and without this change, you'll see in the output how without this change the list of files is huge, but with this change the list of files is much shorter, correctly limited to only the source files for the package.

---

Seems like this is working. Tested in remote branch: https://github.com/microsoft/FluidFramework/pull/17471